netlify.toml

```diff
@@ -1,5 +1,10 @@
 [build.environment]
 NODE_VERSION = "18"
+NETLIFY_NEXT_PLUGIN_SKIP = "true"
```

Comment:
Required so that Netlify doesn't run the Next.js build plugin since our app is a Next.js app. Without this, the build fails to deploy the static one page downtime.html